tests/test_example_dags.py

```diff
@@ -35,6 +35,7 @@ def session():
         "example_append",
         "example_load_file",
         "example_merge_bigquery",
+        "example_dynamic_task_template",
```

Comment:
Btw this test will not succeed on Airflow 2.2.5 job as Dynamic Task Mapping was added in 2.3 @pankajastro

Reply:
How do we handle these cases? 🤔 Should we add a version there and only if this version is installed run the test?

Reply:
Added a new test case let see if it work e04ab283b439b16247bd81bcc240f432c3a19fb8